src/Core/Framework/Test/DataAbstractionLayer/Write/TranslationTest.php

```diff
@@ -589,6 +589,14 @@ public function testUpsert(): void
                 'minPurchase' => 1,
                 'shippingFree' => false,
                 'purchasePrice' => 0,
```

Comment:
leave this out. 1. Checks if trigger works and 2. will not be forgotten when purchasePrice will be removed.